src/pages/createPost.tsx

```diff
@@ -47,229 +75,305 @@ const createPost = () => {
+              </LocalizationProvider>
+              {isEndDateErr && <p style={{ color: "red" }}>{endDateErrMsg}</p>}
+              <div style={helperTextBox}>
+                <FormHelperText sx={helperTextError}>
+                  ช่องนี้ไม่สามารถเว้นว่างได้
+                </FormHelperText>
+              </div>
+            </Grid>
           </Grid>
-
-
-          <Grid item>
-            <Typography variant="h5">Tag</Typography>
-            <Autocomplete
+        </Box>
+        <Box sx={createPostLayout}>
+          <Typography variant="h5" margin="0 0 1vh 0">
+            Tag
+          </Typography>
+          <Autocomplete
             multiple
-            limitTags={5}
-            id="multiple-limit-tags"
             options={Tags}
-            getOptionLabel={(option) => option.Tag}
+            value={selectedTags}
+            onChange={handleAddTag}
+            renderTags={(value: readonly string[], getTagProps) =>
+              value.map((option: string, index: number) => (
+                <Paper
+                  style={{
+                    display: "flex",
+                    justifyContent: "center",
+                    flexWrap: "wrap",
+                    border: "1px solid rgba(0, 0, 0, 0.12)",
+                    borderRadius: "4px",
+                  }}
+                >
+                  <Chip
+                    variant="outlined"
+                    label={option}
+                    style={{ color: "#6200EE", border: "none" }}
+                    {...getTagProps({ index })}
+                  />
+                </Paper>
+              ))
+            }
             renderInput={(params) => (
-              <TextField {...params} placeholder="Click for add tag" fullWidth />
+              <TextField
+                {...params}
+                placeholder="คลิกเพื่อเลือก Tags (เลือกได้สูงสุด 5 Tags)"
+                fullWidth
+              />
             )}
-            
-            />
-            
-            
-          </Grid>
-          
+          />
+          {isSubmit && isTagErr && <p style={{ color: "red" }}>{tagErrMsg}</p>}
+        </Box>
+        <Box sx={createPostLayout}>
+          <Typography variant="h5" margin="0 0 1vh 0">
+            Description
+          </Typography>
+          <TextField
+            defaultValue={initialValue.description}
+            variant="outlined"
+            fullWidth
+            multiline
+            rows={4}
+            size="medium"
+            placeholder="เช่น มาเที่ยวกันเลย ร้านบอร์ดเกมแถวรัชดา"
+            value={desc}
+            onChange={handleDescChange}
+            inputProps={{ maxLength: 100 }}
+          />
+          {isSubmit && isDescErrMsg && (
+            <Box display="flex">
+              <FormHelperText error>{descErrMsg}</FormHelperText>
+            </Box>
+          )}
+          <div style={helperTextBox}>
+            <FormHelperText sx={helperText}>{desc.length}/500</FormHelperText>
```

Comment:
The char limit still at 100 instead of 500. Please fix it.

Reply:
This is for description.
limit at 500

Reply:
what i mean is that right now it was fixed at 100 so you should fix it to be 500

Reply:
try it you will see that you can't type more than 100

Reply:
Okay i see